azure-pipelines.yml

```diff
@@ -0,0 +1,90 @@
+parameters:
+- name: module_names
+  type: object
+  default:
+  - nidigital
+  - nitclk  
+  - nifgen
+  - nidcpower
+  - nidmm
+  - niscope
+  - nimodinst
+  - nise
+  - niswitch
+
+# A pipeline with no CI trigger
+trigger: none
+
+# specific path
+pr:
+  branches:
+    include:
+    - master
+    - releases/*
+  paths:
+    exclude:
+    - CHANGELOG.md
+    - CONTRIBUTING.md
+    - .gitattributes
+    - LICENSE
+    - VERSION
+  drafts: false
+
+stages:
+
+  - stage: ensure_code_gen_up_to_date
```

Comment:
This stage is not described in the Description of the PR.
Either describe it, or (even better) make that change separately. Smaller PRs are always better.